server/mergin/sync/public_api.yaml

```diff
@@ -272,7 +272,7 @@ paths:
         "422":
           description: Project access update could not be processed
           content:
-            application/json:
+            application/problem+json:
```

Comment:
let's grep all yaml files (even in EE/cloud)

Reply:
New commit for the remaining CE `422` custom error response: https://github.com/MerginMaps/server/pull/264/commits/3906205eccb96d08f52d387803efeb9566b2ed12

Workspace yaml: https://github.com/MerginMaps/server-private/pull/2485
Service has no `422` error responses.